tasks/lzards-backup/src/index.ts

```diff
@@ -156,18 +160,23 @@ export const postRequestToLzards = async (params: {
     granuleId,
     provider,
     createdAt,
+    lzardsProvider,
   } = params;
 
-  const lzardsProvider = getRequiredEnvVar('lzards_provider');
+  const configuredLzardsProvider = getLzardsProviderOrDefault(lzardsProvider);
+  if (!configuredLzardsProvider) {
+    log.warn(
+      'Warning - no LZARDS provider set in the configuration object or Cumulus `lzards_provider` configuration.  Backup may fail.'
```

Comment:
Wouldn't the backup _definitely_ fail if there's no provider? Asking if we should throw an error here instead of a log 🤔

Reply:
Good question - to some degree that's lzards's decision.    I considered that behavior, however that makes it a change from the current behavior (the TF configuration currently sets `""` by default)

```
> typeof ""
'string'
```

Open to pushback, of course.

Reply:
Oh sure. I buy that.